### PR TITLE
Add openshift deployment files for react agent,streamlit

### PR DESCRIPTION
**Pull request by hemajv**
(Addresses #2)

This PR adds OpenShift templates to deploy the ReAct agent server and Streamlit web app. The PR also modifies the code to update logging of the agent's outputs to a hosted MLFlow instance rather than a local instance.

### CODE REVIEW

File: react_agent/openshift/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- OpenShift CLI (`oc`) installed and configured
+- (**Optional**) `kubectl` and `kustomize` installed and configured
+
+## Deployment Instructions
+
+### Step 1: Clone the Repository
+
+```sh
+git clone git@github.com:redhat-et/llm-agents.git
+cd react_agent
```
Comment (suppathak) [inline react_agent/openshift/README.md]:
```suggestion
cd llm-agents/react_agent
```

File: streamlit/openshift/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- OpenShift CLI (`oc`) installed and configured
+- (**Optional**) `kubectl` and `kustomize` installed and configured
+
+## Deployment Instructions
+
+### Step 1: Clone the Repository
+
+```sh
+git clone git@github.com:redhat-et/llm-agents.git
+cd streamlit
```
Comment (suppathak) [inline streamlit/openshift/README.md]:
```suggestion
cd llm-agents/streamlit
```

**Review by suppathak — COMMENTED** [review]
Thank you Hema. Added minor comments. Otherwise lgtm.👍🏼

File: react_agent/openshift/README.md (hunk 1 above)
Comment (hemajv) [reply react_agent/openshift/README.md]: good catch 👍

**Comment by hemajv** [comment]
> Thank you Hema. Added minor comments. Otherwise lgtm.👍🏼

Thanks @suppathak, updated the PR!

**Review by suppathak — APPROVED** [review]

File: react_agent/openshift/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```
Comment (Shreyanand) [inline react_agent/openshift/configmap.yaml:1]: Minor nit that can be addressed in a subsequent PR:
Can we add in the readme that the tool definition has to be updated in this configmap?

**Review by Shreyanand — APPROVED** [review]
LGTM 👍

**Pull request merged**